### ISSUE TITLE
Autofill in android does not identify url port

### ISSUE THREAD

**Issue by c9DM8veydk5eEPsWAUxpJs7PtPNlaEPsrY7BUMT** [bitwarden/mobile#2970]
### Steps To Reproduce

1. Open a single word local domain (e.g. server:8080) in any browser
2. Click to auto fill pop up
3. App opens and site name in app is -- instead of site url

### Expected Result

App opens and present password for server:8080

### Actual Result

App opens and site name in app is -- instead of site url and no passwords are returned

### Operating System

Android

### Operating System Version

Android 14

### Build Version

2024.1.1

### Beta

- [ ] Using a pre-release version of the application.

**Comment by uniquePWD** [comment]
This is an issue in the beta too.

**Comment by houruomu** [comment]
Yes, the issue persists in the current version. I tried "host" mode, which works fine on PC but not on android.

**Comment by closebot-bw** [comment]
This repo has been archived. To report an issue you're experiencing in the new native apps, visit the [iOS](https://github.com/bitwarden/ios) / [Android](https://github.com/bitwarden/android) repo.
